### PR TITLE
Sinon

### PR DESCRIPTION
**Pull request by reecemillsom**
#### What does this PR do? (please provide any background)

This PR adds the sinon.sandbox details into the testing standards. In the url when reviewing the files changed can you put ?w=1 then will only see what i have added
#### What tests does this PR have?

N/A
#### How can this be tested?

N/A
#### Screenshots / Screencast

N/A
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [x] I have checked the [contributing document](../blob/master/CONTRIBUTING.md) and I'm happy for this to be reviewed.

I have added this into the testing-standard and would like to know what people think of it.

### CODE REVIEW

File: testing-standard.md [hunk 1]
```diff
@@ -177,10 +177,42 @@ There are three things we should check for in the above function:
...
+By stubbing `getHoursAtBirth` and `getHoursAtDeath` we can **a)** check that both of these are called (ticking off points 1 and 2 above), and **b)** ensure that the real versions of `getHoursAtBirth` and `getHoursAtDeath` are never actually called. This is crucially important as we're **not testing that those functions work**. We only care that they've been called and that what we do in the `hoursAlive` function works.
 
 Other tests will check that `getHoursAtBirth` and `getHoursAtDeath` work, and by building up a suite of tests that ensure each function works as it's supposed to, we can build confidence that our codebase works correctly. It's for this reason that `stubs` are preferred to spies in our unit tests - we don't want to be running code outside of the function we're actually testing during a unit test.
 
+### Sinon Sandbox
+
+Using **sinon.sandbox** object to hold your stubs and spies.
```
Comment (DuncanFenning) [inline testing-standard.md]: You could just go straight into benefits i.e

`Using sinon.sandbox object to hold your stubs and spies allows you to be able to put ... etc`

File: testing-standard.md [hunk 2]
```diff
@@ -177,10 +177,42 @@ There are three things we should check for in the above function:
...
+
+Benefits:
+
+Allows you to be able to put multiple stubs within an object when creating them in a **beforeEach** and instead of restoring them one at a time can restore them all at once.
+This is nice because it means we can restore all of our stubs at once and removes a few lines of code.
```
Comment (DuncanFenning) [inline testing-standard.md]: Maybe - 

'instead of restoring them one at a time, you can restore them all at once'

**Comment by robhuzzey** [comment]
This is a very good find! :+1:

File: testing-standard.md [hunk 3]
```diff
@@ -177,10 +177,42 @@ There are three things we should check for in the above function:
...
+
+Code:
+Put the creation of your sandbox before the beforeEach because then your **afterEach** has scope to what is in your **beforeEach**.
+
```
Comment (DuncanFenning) [inline testing-standard.md:187]: don't think you need the beforeEach/afterEach explanation as devs already know this.

Same as on line 214

**Comment by DuncanFenning** [comment]
Like this :+1:

**Comment by notABluesSinger** [comment]
Nice find :+1: :shipit:

**Comment by reecemillsom** [comment]
Thanks people :)

**Pull request merged**